source/user-manual/capabilities/wazuh-logtest/index.rst

```diff
@@ -0,0 +1,23 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. _manual_wazuh_logtest:
+
+Wazuh-Logtest
+=============
+
+.. versionadded:: 4.1.0
+
+The Wazuh-Logtest whole solution was designed to replace :ref:`ossec-logtest tool <ossec-logtest>`,
+featuring to test and verify the ruleset locally and remotely, sharing the rules engine 
```

Comment:
change featuring *to* ... for : now allowing  also to test and verify rules and decoders remotely